.eslintrc.json

```diff
@@ -32,7 +28,9 @@
         "@typescript-eslint/no-empty-interface": [
           "error",
           {"allowSingleExtends": true}
-        ]
+        ],
+        "node/no-unsupported-features/node-builtins": "off",
```

Comment:
apparently this is deprecated and it wasn't properly using our defined minimum node version. eslint was mad at me becuase `URL` is node >= 10 which should be ok.

Reply:
instead I think we should make sure that it ESLint the correct version